util/test/demos/vk/vk_shader_debug_zoo.cpp

```diff
@@ -1576,6 +1578,17 @@ void main()
       Color += vec4(1.0, 1.0, 1.0, 1.0);
       break;
     }
+    case 177:
+    {
+      Color = vec4(imageLoad(texBufferUint, int(zeroi+2)).x, 1, 0, 1);
+      break;
+    }
+    case 178:
```

Comment:
Thank you for adding this test : it does fail before the code change and passes after the code change